docs/guides/ssr.md

```diff
@@ -58,10 +58,10 @@ To support caching queries on the server and set up hydration:
 - Wrap your app component with `<Hydrate>` and pass it the `dehydratedState` prop from `pageProps`
 
 ```tsx
-// _app.jsx
+// _app.tsx
 import { Hydrate, QueryClient, QueryClientProvider } from '@tanstack/react-query'
 
-export default function MyApp({ Component, pageProps }) {
+export default function MyApp({ Component, pageProps }: AppProps<any>) {
```

Comment:
```suggestion
export default function MyApp({ Component, pageProps }: AppProps<{ dehydratedState: unknown }>) {
```

would work, but I'm not sure if we'd want to have docs in TS for just one page. If we do this, it should probably be for all examples of the documentation at the same time. This is also not specific to react-query - it's a nextjs typings "problem".